service/create/secrets.go

```diff
@@ -0,0 +1,72 @@
+package create
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	microerror "github.com/giantswarm/microkit/error"
+	"k8s.io/client-go/pkg/api"
+	apierrs "k8s.io/client-go/pkg/api/errors"
+	"k8s.io/client-go/pkg/api/v1"
+	"k8s.io/client-go/pkg/watch"
+)
+
+const (
+	componentLabel string        = "clusterComponent"
+	clusterIDLabel string        = "clusterID"
```

Comment:
IMO these magic strings should be in a common place where operators and the TPO creator can both read it from. I think a good place would be the TPR repos.